packages/vscode-extension/src/contextService.ts

```diff
@@ -0,0 +1,31 @@
+import { Neo4jSettings } from '@neo4j-cypher/language-server/src/types';
+import { ExtensionContext } from 'vscode';
+
+type Neo4jLanguageClient = {
+  sendNotification: (method: string, settings: Neo4jSettings) => Promise<void>;
+};
```

Comment:
there might be a better way to achieve this.. but I defined this type so I could mock/stub the functionality of the LanguageClient more easily 

creating a mock object that extended the LanguageClient was proving troublesome.. there was some internal initialization that was tricky to provide dummy values for.. and since we currently only use one function of it in the extension.. sendNotification.. creating this type and using it in place of the LanguageClient for parameter types makes it a whole lot easier to stub